cpp/src/parquet/xxhasher.h

```diff
@@ -29,12 +25,10 @@
 
 namespace parquet {
 
-/// Source:
-/// https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp
-/// (Modified to adapt to coding conventions and to inherit the Hasher abstract class)
-class PARQUET_EXPORT MurmurHash3 : public Hasher {
+class PARQUET_EXPORT XxHasher : public Hasher {
  public:
-  MurmurHash3() : seed_(DEFAULT_SEED) {}
+  XxHasher() : seed_(kDefaultSeed) {}
```

Comment:
Just for the record, is the seed parameter actually useful? The Parquet spec mandates seed value 0. Hardcoding the seed might make hashing slightly faster.

Reply:
Seems that release will optimize this. I just keep like previous MurmurHash3, since `xxhasher` and it's base class "hasher" is not `parquet_xxhasher`.

Reply:
https://godbolt.org/z/bjz5WGx83 Seems maybe it will. Still I think `Hasher` is not ParquetHasher, the name of it is too

Reply:
> https://godbolt.org/z/bjz5WGx83 Seems maybe it will.

I don't think this example is proving anything as 1) everything is implemented in the  `.h` (thus trivially inlinable) 2) the result value doesn't even depend on the seed.

> Still I think `Hasher` is not ParquetHasher, the name of it is too

No, but it's in the `src/parquet` directory, which is a good hint :-)